test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,89 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('First Api Tests', () => {
+});
+
+it('Consume GET Service', async () => {
```

Comment:
The idea of ITs is to have them inside the describe, it is a section. (Same for the other its)